lighthouse-core/test/lib/tracehouse/trace-processor-test.js

```diff
@@ -20,6 +20,7 @@ const noFCPtrace = require('../../fixtures/traces/airhorner_no_fcp.json');
 const noNavStartTrace = require('../../fixtures/traces/no_navstart_event.json');
 const backgroundTabTrace = require('../../fixtures/traces/backgrounded-tab-missing-paints.json');
 const lcpTrace = require('../../fixtures/traces/lcp-m78.json');
+const lcpAllFramesTrace = require('../../fixtures/traces/lcp-all-frames-m89.json');
```

Comment:
these seem a little too big, were they run through the minifier?

Reply:
https://github.com/GoogleChrome/lighthouse/blob/4651b52b19780cab2fb9e58cb34a2cb18468fa05/lighthouse-core/lib/minify-trace.js#L1

Reply:
`yarn minify-latest-run` if you just ran -G :)

OR

`node lighthouse-core/scripts/lantern/minify-trace.js <trace in> <trace out>`
`node lighthouse-core/scripts/lantern/minify-devtoolslog.js <log in> <log out>`

Reply:
They were run through the minifier...

Reply:
Yikes and still 17000 😬 was this a sample iframe'd page you're using?

Reply:
Uhhh no, it was espn.com 😬 . Maybe a simple iframe test would be better.

Reply:
ahhhh it all makes sense now 😆 

yeah if all we're looking for is this, I'd vote to keep the fixtures simple and easy to understand. maybe combine the CLS/LCP examples linked in the framehole doc?